File: Section1 - Basics/Patterns/pattern17.js
function printTriangle(n) {
	let str = "";

	for (let i = 0; i < n; i++) {
		let x = 65;
		for (let j = n; j > i + 1; j--) {
			str = str + " ";
		}
		for (let k = 0; k <= i; k++) {
			str = str + String.fromCharCode(x);
			x++;
		}
		x = 65;
		let s = "";
		for (let j = 0; j < i; j++) {
			s = String.fromCharCode(x) + s;
			x++;
		}
		str = str + s;
		str += "\n";
	}
	console.log(str);
}

printTriangle(5);

File: Section1 - Basics/Patterns/pattern17.java
void printTriangle(int n) {
    // code here
    for(int i=0; i<n; i++) {
        int x=65;
        for(int j=n; j>i+1; j--) {
            System.out.print(" ");
        }
        for(int k=0; k<=i; k++) {
            System.out.print((char)x);
            x++;
        }
        x=65;
        String s = "";
        for(int j=0; j<i; j++) {
            s= (char)x+ s;
            x++;
        }
        System.out.print(s);
        System.out.print("\n");
    }
}